Dockerfile

```diff
@@ -62,6 +63,7 @@ RUN ARCH="$(uname -m)" && \
 
 # kubectl install
 ARG KUBECTL_VERSION
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
```

Comment:
```suggestion
```

Duplicates the line 23 - ref. https://docs.docker.com/reference/dockerfile/#shell